docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -266,3 +266,35 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *processes* data that may be PIIs or sensitive:
+
+### Processing
```

Comment:
I'd get rid of this header. It repeats what the header directly above it says, and it's the only section here.